wiki/Getting_Started_Contributing_to_Wiki/en.md

```diff
@@ -0,0 +1,164 @@
+
+## Forking
+ 
+ To be able to make changes to the repository, you must first create your own copy of the repository. This is known as "forking".
+ 
+ Go to the [osu!wiki repository](https://github.com/ppy/osu-wiki).
+ 
+ You should see something like this.
+
+![osu-wiki Repository Snapshot](3.JPG "Snapshot of the osu!Wiki Repository")
+
+Click on the "Fork" button, located at the top-right of the page. This should redirect you to the location for the fork.
+
+![Fork prompt dialogue](4.JPG "Fork prompt dialogue")
+
+Now click at your profile and GitHub would redirect you to where the fork is.
+
+Finally, you should see something like this.
+
+![Fork Snapshot](5.JPG "Fork Snapshot")
+
+Congratulations! You have made a copy of the osu!wiki repository! You can now proceed to make changes.
+## Making Changes 
+
+### Through GitHub's Web Editor
+If you don't want the hassle of cloning your fork into your PC, GitHub offers editing through your browser.
+
+To edit a page, go to the folder of that page and select the language of that article. From here, you should see something like this.
+
+_Note: Each article is grouped into folders where they contain markdown files arranged by language and their media. All article pages inside these folders must use the two letter language codes. To know which language code to use, see the [List of ISO 639-1 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) following the `639-1` column. Please follow this format when creating a new article._                  
+
+![GitHub Preview](7.JPG "Github Preview")
+
+Click on the pencil icon to edit; this should redirect you to the web editor.
+
+![GitHub's Web Editor](8.JPG "web Editor Snapshot")
+
+If you want to preview your changes, click the "Preview Changes" tab. To go back and edit, click the "Edit File" tab.
+
+After making your changes, scroll down and fill the following fields to explain your edit(s) to other contributors.
+
+![Commit Message](9.JPG "Commit Message")
+
+_**Note: Your commit message(s) must thoroughly explain your changes, otherwise, they may not get merged!**_
+
+You would be given an option where to save your edits.
+
+- **"Commit Directly to the ``xyz`` branch"** 
+  - This will commit the changes through your branch directly.
+
+- **"Create a New Branch"** 
+  - Automatically creates a Branch for your edits. This is useful if you're making bulk edits.
+
+Pick an option and click "Commit Changes".
+
+### Through your Local Machine
+
+GitHub under the hood is a Git Service. You can clone your fork to your PC and make changes there.
+
+if you have Git installed, enter this in your Command Line Interface (CLI):
+```prolog
+git clone https://github.com/<your github username (ie. peppy)>/osu-wiki 
+```
+*Note: If you don't have Git installed, you can go to [their website](https://git-scm.com/) to download Git.*
+
+and this should clone your fork into the folder you specified. You can now proceed to make your changes.
```

Comment:
cloning is more straightforward, as it doesn't require you to init a empty git dir and IMHO the fork should be done on web